src/app/docs/button/button-docs.component.html

```diff
@@ -0,0 +1,196 @@
+<sky-docs-demo-page
+  pageTitle="Buttons"
+>
+  <sky-docs-demo-page-summary>
+    The button classes create buttons to trigger actions from within an interface. For information about styles and implementation, see the <a href="https://developer.blackbaud.com/skyux/design/guidelines/buttons-links">buttons and links guidelines</a>.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-design-guidelines>
+
+    <sky-docs-demo-page-section
+      heading="Button classes"
+    >
+      <table class="sky-docs-table">
+        <tr>
+          <td class="sky-docs-table-cell">
+            <button type="button" class="sky-btn sky-btn-primary">Primary</button>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Usage
```

Comment:
Do we really need these **Usage** and **Code** paragraphs in each row? Seems to me like a pattern that we could drop as we migrate over. If we really need anything, we could add column headers at the top of the whole table instead of on a row-by-row basis, but I don't really think we need that.

Reply:
I took them out, and the table looks much neater.